src/pocketdb/web/SearchRpc.cpp

```diff
@@ -334,6 +345,15 @@ namespace PocketWeb::PocketWebRpc
         int depth = 1000;
         int cntOut = 10;
 
+        if (request.params.size() > 2 && request.params[2].isNum() && request.params[2].get_int() > 0)
```

Comment:
Same comment as function above, requrest.params.size(), return error if we get no parameters or too many.